hw/ip/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -211,12 +211,17 @@ num_grps = len(grps)
 
     { name: "CLK_HINTS",
       desc: '''
-        Clock hint for software gateable clocks.
-        These clocks are not fully controlled by software.
+        Clock hint for software gateable transactional clocks during active mode.
+        During low power mode, all clocks are gated off regardless of the software hint.
 
-        For disable, software only provides a hint, and hardware determines the final clock state based on the
-        hint and whether the block in question is idle.
+        Transactional clocks are not fully controlled by software.  Instead software provides only a disable hint.
 
+        When software provides a disable hint, the clock manager checks to see if the associated hardware block is idle.
+        If the hardware block is idle, then the clock is disabled.
+        If the hardware block is not idle, the clock is kept on.
+
+        For the enable case, the software hint is immediately honored and the clock turned on.  Hardware does not provide any
+        feedback in this case.
```

Comment:
```suggestion
        While the disable hint is active, the clock manager waits until the associated hardware block is idle before disabling the clock.
        When the software hint becomes inactive the clock is turned on immediately,since the block is idle while its clock is off.
```